tensorflow_addons/layers/normalizations_test.py

```diff
@@ -184,7 +187,7 @@ def test_axis_error(self):
         with self.assertRaises(ValueError):
             GroupNormalization(axis=0)
 
-    @test_utils.run_in_graph_and_eager_modes
+    # @test_utils.run_in_graph_and_eager_modes
```

Comment:
Hmmm why these commented out decorators below the decorated Class?

Reply:
ditto for the below ~4 other cases of these comments

Reply:
Thanks @seanpmorgan. Updated.